gradle.properties

```diff
@@ -4,6 +4,7 @@ group=org.jetbrains.kotlinx
 kotlin_version=2.1.0
 # DO NOT rename this property without adapting kotlinx.train build chain:
 atomicfu_version=0.26.1
+benchmarks_version=0.4.13
```

Comment:
It might be worth plugging `benchmarks_jmh_version=1.37` here, otherwise benchmarks will use pretty ancient version of JMH (1.21).